python/setup.py

```diff
@@ -219,6 +219,10 @@ def get_packages(self):
         ],
     }
 
+    if sys.version_info >= (3, 7):
+        # Numpy dropped python 3.6 support in 1.20.
+        setup_spec.extras["data"].append("numpy >= 1.20")
```

Comment:
I'm wondering what's the difference between data extras and https://github.com/ray-project/ray/blob/master/python/requirements/data_processing/requirements_dataset.txt

Reply:
Data extras is what's actually installed when you do `pip install ray[data]`. Those requirements files should be for additional ray developer add-ons.